### ISSUE TITLE
[FR] Eliminate dependency on statsmodels

### ISSUE THREAD

**Issue by jruedai08** [ligon/Metrics_Miscellany#2]
I think it's a mistake to have an OLS function which depends on statsmodels。  This is a very heavy-weight dependency!

**Comment by ligon** [comment]
Agreed!  Partly addressed by pulling DataMat out of MetricsMiscellany.
